### PR TITLE
add docs site & ci

### PR DESCRIPTION
**Pull request by ofek**
@maennchen

### CODE REVIEW

File: docs/static_provisioning.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+You can pass arbitrary flags to [gcsfuse](https://github.com/GoogleCloudPlatform/gcsfuse) by setting
+`flags` in `volumeAttributes` e.g. `--limit-ops-per-sec=10 --only-dir=some/nested/folder`.
+
+## Permission
+
+In order to access anything stored in GCS, you will need [service accounts][gcp-service-account] with
+appropriate IAM roles. If writes are needed, you will usually select `roles/storage.objectAdmin` scoped
+to the desired buckets.
+
+The [easiest way][gcp-create-service-account] to create service account keys, if you don't yet
```
Comment (maennchen) [inline docs/static_provisioning.md:106]: This link doesn’t work.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,30 +0,0 @@
-# Contributing
```
Comment (maennchen) [inline CONTRIBUTING.md:1]: We should keep the file and link to the docs page.

File: docs/contributing/setup.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* Create Secret
+    - `kubectl create secret generic csi-gcs-secret --from-file=key=service-account.json`
+* Pull Needed Images
+    - `docker pull quay.io/k8scsi/csi-node-driver-registrar:v1.2.0`
+* Apply config `kubectl apply -k deploy/overlays/dev`
+
+## Rebuild & Test
+
+* Build Docker Image `invoke image`
+* Delete Pod
```
Comment (maennchen) [inline docs/contributing/setup.md:32]: We should mention how to build and view the docs.

**Comment by ofek** [comment]
@maennchen addressed!

**Comment by maennchen** [comment]
@ofek LGTM

**Comment by maennchen** [comment]
@ofek Oh, two more improvements:

Add status badge to readme:

```
![Test](https://github.com/ofek/csi-gcs/workflows/Test/badge.svg)
```

Add [`upload-artifact`](https://github.com/actions/upload-artifact) to workflow for docs.

**Comment by ofek** [comment]
Can you try artifacts later?

**Pull request merged**